examples/complex_example.py

```diff
@@ -1,21 +1,24 @@
+from graphene import InputObjectType, Float, ObjectType, String, Field, Schema
+
+# Naming confusion/conflict with class Mutation and explicit import of graphene.mutation.
```

Comment:
I'd love some thoughts on this code. We're importing `Mutation` from `graphene`, previously used as `graphene.Mutation`, _and_ declaring `class Mutation`. 

Side note: We need to do a run through of all the doc examples in a functional context.

Reply:
Yeah, that's no good. This example will need to be improved.

Reply:
And that is exactly why you should just `import graphene` so each external dependency is always clearly distinguishable as external.